.travis.yml

```diff
@@ -35,6 +35,7 @@ matrix:
             - sourceline: 'deb https://deb.torproject.org/torproject.org tor-nightly-0.2.9.x-xenial main'
               key_url: 'https://deb.torproject.org/torproject.org/A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89.asc'
           packages:
+            - shellcheck
```

Comment:
I think this is not necessary, as Travis has shellcheck pre-installed now. See: https://github.com/koalaman/shellcheck#travis-ci

Reply:
shellcheck is not installed via dpkg in our xenial instances:
https://travis-ci.org/torproject/chutney/jobs/517584623#L1230

Even if it is installed, it's good practice for us to list our dependencies. It helps document what we need, and it insulates us from future travis changes. (Travis have removed default packages before, in order to reduce image size.)